packages/veritone-react-common/src/components/Parts/Buttons/PillButton/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import defaultStyles from './styles.scss';
+
+export default class PillButton extends Component {
+	static propTypes = {
+		label: PropTypes.string,
+		info: PropTypes.string,
+		style: PropTypes.instanceOf(Object),
+		gapStyle: PropTypes.instanceOf(Object),
+		labelStyle: PropTypes.instanceOf(Object),
+		infoStyle: PropTypes.instanceOf(Object),
```

Comment:
is there a strong reason to support both overriding with both inline styles _and_ classnames? I'd prefer to not have inline styles at all if possible